### PR TITLE
Fix Translateable typo, and ease usage of translation replacements

### PR DESCRIPTION
**Pull request by ExE-Boss**

### CODE REVIEW

**Comment by RX14** [comment]
An example of usage?

**Comment by ExE-Boss** [comment]
An example is in my testing block and item I use to test #252.
I will push that soon as a new PR for NOVA-Example.

File: src/test/java/nova/core/language/TranslateableTest.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		public String getUnlocalizedName() {
+			return "$filledStr $emptyStr";
+		}
+
+		@Override
+		public String getLocalizedName() {
+			return languageManager.translate(this.getUnlocalizedName(), this.getReplacements());
+		}
+	}
+}
```
Comment (RX14) [inline src/test/java/nova/core/language/TranslateableTest.java]: No newline at the end of the file. Please make your IDE enforce this.

Comment (ExE-Boss) [inline src/test/java/nova/core/language/TranslateableTest.java]: I accidentally removed that newline in my IDE’s unit test template file

**Pull request merged**